config/management/src/lib.rs

```diff
@@ -113,7 +124,13 @@ impl Command {
             Command::InsertWaypoint(_) => self.insert_waypoint().unwrap().to_string(),
             Command::OperatorKey(_) => self.operator_key().unwrap().to_string(),
             Command::OwnerKey(_) => self.owner_key().unwrap().to_string(),
+            Command::ReadAccountState(_) => format!("{:?}", self.read_account_state().unwrap()),
             Command::SetLayout(_) => self.set_layout().unwrap().to_string(),
+            Command::SubmitTransaction(_) => self
+                .submit_transaction()
```

Comment:
Do you think it makes sense to have "submit_transaction()" return the Transaction that was submitted? This might also be useful for debugging/testing (i.e., the transaction is assumed to be in file, in lcs encoded format). Printing out the transaction that was submitted might provide the user with some assurances about what was submitted? It would also avoid having to define an arbitrary "success" message 😄

Reply:
Hmm... seems an interesting proposal... we actually need to distinguish verbosity here... something to think about..

Yes, I did test this lol on pre-mainnet -- the one thing missing is the ability to query the outcome of a transaction